src/metaschema/oscal_metadata_metaschema.xml

```diff
@@ -54,7 +54,12 @@
                         <enum value="alternate">The link identifies an alternative location or format for this resource. Defined by <a href="https://html.spec.whatwg.org/multipage/links.html#linkTypes">the HTML Living Standard</a></enum>
                         <enum value="predecessor-version">This link identifies a resource containing the predecessor version in the version history. Defined by  <a href="https://tools.ietf.org/html/rfc5829">RFC 5829</a>.</enum>
                         <enum value="successor-version">This link identifies a resource containing the predecessor version in the version history. Defined by <a href="https://tools.ietf.org/html/rfc5829">RFC 5829</a>.</enum>
+                        <enum value="version-history">This link identifies a resource containing the version history of this document. Defined by <a href="https://tools.ietf.org/html/rfc5829">RFC 5829</a>.</enum>
```

Comment:
Just skimmed RFC 5820. Seems like a minor value add that would let people point to a local or web-hosted VCS tool (gitweb, GitHub, even subversion) and is a non-breaking change.

Reply:
Ok. This is fine.